oamapps/columnstoreSupport/configReport.sh

```diff
@@ -46,10 +42,10 @@ if [ -n "$systemctl" ]; then
 	echo " "
 	echo "################# systemctl list-unit-files --type=service | grep columnstore #################"
 	echo " "
-	$SUDO systemctl list-unit-files --type=service | grep columnstore 2>/dev/null
+	systemctl list-unit-files --type=service | grep columnstore 2>/dev/null
 	echo "################# systemctl list-unit-files --type=service | grep mysql-Columnstore #################"
 	echo " "
-	$SUDO systemctl list-unit-files --type=service | grep mysql-Columnstore 2>/dev/null
```

Comment:
This is interesting.  That systemctl cmd doesn't need root perms; not sure why sudo was used for this.